src/components/fileStorage/repo/files.repo.integration.test.js

```diff
@@ -41,7 +41,7 @@ describe('files.repo.integration.test', () => {
 			let file;
 
 			beforeEach(async () => {
-				fileOwnerId = generateObjectId();
+				fileOwnerId = new generateObjectId();
```

Comment:
Could you remove this `generateObjectId` function from testObjects and just import `ObjectId` directly in the tests where needed? I think it looks weird to new a function.

Reply:
Could we maybe write a helper function like `generateObjectId` in `testObjects` that would include the `new ObjectId()`?

Reply:
But why not import ObjectId directly? We do this in other tests, too, like src/components/user/repo/user.repo.integration.test.js

Reply:
Would be less changes? But yes, I agree. That would be the best solution.